wcfsetup/install/files/js/WoltLabSuite/Core/Prism.js

```diff
@@ -0,0 +1,59 @@
+/**
+ * Augments the Prism syntax highlighter with additional functions.
+ * 
+ * @author	Tim Duesterhus
+ * @copyright	2001-2018 WoltLab GmbH
+ * @license	GNU Lesser General Public License <http://opensource.org/licenses/lgpl-license.php>
+ * @module	WoltLabSuite/Core/Prism
+ */
+
+window.Prism = window.Prism || {}
+window.Prism.manual = true
+
+define(['prism/prism'], function () {
+	Prism.wscSplitIntoLines = function (container) {
+		var frag = new DocumentFragment();
```

Comment:
The `DocumentFragment` constructor is still “experimental” and unsupported by IE11.
```suggestion
		var frag = document.createDocumentFragment();
```